src/pymmcore_widgets/mda/_core_mda.py

```diff
@@ -90,6 +110,21 @@ def __init__(
         self.save_info.valueChanged.connect(self.valueChanged)
         self.control_btns = _MDAControlButtons(self._mmc, self)
 
+        # add HCS wizard
```

Comment:
break out integration of the HCS wizard with the MDAWidget (and associated tests) into a new PR